closure/testing/closure_js_test.bzl

```diff
@@ -104,6 +105,7 @@ _closure_js_test = rule(
         "pedantic": attr.bool(default=False),
         "defs": attr.string_list(),
         "_closure_library_base": CLOSURE_LIBRARY_BASE_ATTR,
+        "_closure_library_deps": CLOSURE_LIBRARY_DEPS_ATTR,
```

Comment:
True, however "bazel test closure/..." fails without this line as `collect_js_srcs()` is used by `closure_js_test()`. If you have concerns with this I can modify `collect_js_srcs()` to not include `deps.js` for the testing case however I'm not sure that it is worth adding a (small) special case just to omit `deps.js` from tests.

100% your call. I'll gladly make the change if you like.